ansible/roles/dev-desktop/files/skel/config.toml

```
@@ -0,0 +1,9 @@
+changelog-seen = 2
+
+# Change this to the profile of your choice
+profile = "compiler"
+
+[rust]
+# Build with debug assertions so we get useful RUSTC_LOG
+# and notice debug assertions failing.
+debug = "true"
```

```suggestion
debug = true
```

so we don't get.....

```
failed to parse TOML configuration 'config.toml': invalid type: string "true", expected a boolean for key `rust.debug`
```